crates/oxc_linter/src/module_record.rs

```diff
@@ -282,6 +282,9 @@ impl<'a> From<&oxc_syntax::module_record::ImportImportName<'a>> for ImportImport
 /// ```
 #[derive(Debug, Default, Clone, PartialEq, Eq)]
 pub struct ExportEntry {
+    /// Span of the export statement.
+    pub statement_span: Span,
+
     /// Span for the entire export entry
     pub span: Span,
```

Comment:
I do not know why this property is there and wrong. 
Hope that somebody will look into it and maybe refactor it